interview-questions.md

```diff
@@ -10,58 +10,71 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What is `this` in JavaScript?
```

Comment:
For your interview questions, please add more of your experience with the answers. Tell the story how your challenge questions helped you understand the concept or how you explained it to your pair partner. These interview questions are opportunities to practice sharing your thoughts with an interviewer/future clients.

Reply:
Thanks Charlean, I'll keep this in mind.